python/foglamp/services/south/ingest.py

```diff
@@ -563,6 +563,11 @@ def is_available(cls) -> bool:
 
         list_size = len(readings_list)
 
+        # asset tracker checking
+        payload = {"asset": asset, "event": "Ingest", "service": cls._parent_service._name,
+                   "plugin": cls._parent_service._plugin_handle['plugin']['value']}
+        cls._parent_service._core_microservice_management_client.create_asset_tracker_event(payload)
```

Comment:
can we gave have a last_event here? and when a new event happens and differs then only pass to compare with loaded events and insert as required.

even if loaded events are cached, a last_event still makes sense as it will be  comparison cost with 1 obj vs find in
many loaded events

Reply:
Done